airflow/contrib/hooks/gcp_spanner_hook.py

```diff
@@ -41,11 +42,12 @@ def __init__(self,
     def get_client(self, project_id):
         # type: (str) -> Client
         """
-        Provides a client for interacting with Cloud Spanner API.
+        Provides a client for interacting with the Cloud Spanner API.
 
-        :param project_id: The ID of the project which owns the instances, tables and data.
+        :param project_id: TThe ID of the  GCP project that owns the Cloud Spanner
```

Comment:
looks like a typo.

```suggestion
        :param project_id: The ID of the  GCP project that owns the Cloud Spanner
```

I haven't reviewed entire PR yet!!

Reply:
OK. I will wait with further rebases until you review all :)

Reply:
This is still pending